### DISCUSSION TITLE
Enhancement Suggestion for 'Search through your open tabs' Feature

### DISCUSSION THREAD

**Discussion by nelzomal** [philc/vimium#4366] [General]
I really appreciate the 'Search through your open tabs' feature; it's incredibly useful. However, I've noticed a challenge when dealing with multiple windows. If I have several windows open, each with numerous tabs, locating a specific tab becomes somewhat cumbersome.

In my usual workflow, I tend to open multiple windows, each dedicated to a different topic. I also prefer to name each window for better organization. Given this setup, I believe we could enhance the tab search functionality, especially for users who work with multiple windows.

Here are a few suggestions for improvement:

Prioritize Current Window: Display results from the current window first in the search. This could streamline finding tabs in the active context.
Window-Specific Search: Introduce the ability to search tabs within a specific window. This would be immensely helpful for users who segregate their work into different window topics.
Include Window Information in Results: Show window names or identifiers in the search results. This would provide quick context and make it easier to identify where each tab is located.

**Comment by ispringle** [comment]
I think this would be a great idea as well. One way to realize this would be to allow for specific "filters" on the Vomnibar. For example if you specified something like `Vomnibar.activate.tabs` it would only show open tabs. You could have one for `history` as well
